src/Illuminate/View/Concerns/ManagesLayouts.php

```diff
@@ -146,11 +146,7 @@ protected function extendSection($section, $content)
      */
     public function yieldContent($section, $default = '')
     {
-        $sectionContent = $default instanceof View ? $default : e($default);
-
-        if (isset($this->sections[$section])) {
-            $sectionContent = $this->sections[$section];
-        }
+        $sectionContent = $this->sections[$section] ?? ($default instanceof View ? $default : e($default));
```

Comment:
Maybe still keep the first line separately here
```suggestion
        $sectionContent = $default instanceof View ? $default : e($default);
        $sectionContent = $this->sections[$section] ?? $sectionContent;
```

Reply:
By using in a single line it will avoid the right-side statements to be called. It is a micro-optimization, I know, but I think that the readability is still better in a single line than multi. In addition, this method should not receives too much updates, I believe, so I think that do not have problem to keep it as I did. What do you think?

Reply:
Yeah it's fine.